wrench/src/premultiply.rs

```diff
@@ -6,9 +6,9 @@
 pub fn premultiply(data: &mut [u8]) {
     for pixel in data.chunks_mut(4) {
         let a = pixel[3] as u32;
-        let r = pixel[2] as u32;
+        let b = pixel[2] as u32;
```

Comment:
is this change intentional? it doesn't match the `unpremultiply` code below

Reply:
Yep - without this change the colors in the Firefox image logo I am using in the tests is wrong. I updated ```unpremultiply``` to match.